### PR TITLE
feat: add init-containers to crane

### PR DESCRIPTION
**Pull request by nandor-magyar**
Arbitrary init containers can be defined with this for imports, seeding etc.

### CODE REVIEW

File: web/crux-ui/src/server/crux/mappers/image-mappers.ts [hunk 1]
```diff
@@ -21,6 +21,7 @@ import {
   restartPolicyToJSON,
   Volume,
 } from '@app/models/grpc/protobuf/proto/common'
+import { InitContainer } from './../../../models/grpc/protobuf/proto/common'
```
Comment (robot9706) [inline web/crux-ui/src/server/crux/mappers/image-mappers.ts]: Use @app import.

File: agent/pkg/crane/k8s/deployment.go [hunk 2]
```diff
@@ -81,7 +81,7 @@ func (d *deployment) deployDeployment(p *deploymentParams) error {
 					d.appConfig.KeyIssuer:  p.issuer,
 				}).WithSpec(
 					corev1.PodSpec().WithContainers(containerConfig).
-						WithInitContainers(getInitContainers(p.containerConfig, d.appConfig)...).
+						WithInitContainers(getInitContainers(p.containerConfig, d.appConfig, p)...).
```
Comment (robot9706) [inline agent/pkg/crane/k8s/deployment.go:84]: If `p` is passed to this function then passing `p.containerConfig` could be simplyfied just by passing `p` and then getting `containerConfig` later in the function.

**Review by robot9706 — CHANGES_REQUESTED** [review]

**Review by robot9706 — APPROVED** [review]
LGTM 😎

**Review by c3ppc3pp — APPROVED** [review]
:+1:

**Pull request merged**